styles/styles.css

```diff
@@ -683,6 +683,18 @@ main .section.no-margin-bottom {
   margin-bottom: 0;
 }
 
+/* section - add pidding 1rem */
+main .section.padding-1 {
```

Comment:
There's another PR from @dnbute proposing a padded section. I think we should have a common solution. Maybe add a section metadata property if there's no uniformity.

Reply:
I think jiang's section is more like a less narrow `narrow` section; mine has padding all around

I think we could make this padding (0 1rem) default for all sections and make sure other styles override this, like the `narrow` and `narrower` sections; just my two cents

Reply:
@dnbute @sdmcraft 
Yes. My case only needs padding on the left and right sides. We can make it as default styling but I'm not sure the impact, maybe we need to check all pages. I suggest keeping it as a variation and changing the name to `small-narrower`. What do you think?

>I think jiang's section is more like a less narrow narrow section; mine has padding all around
>I think we could make this padding (0 1rem) default for all sections

Reply:
Visually I don't find this making a lot of difference unless I am missing something. Can we just omit this style?